### PR TITLE
samples: nrf9160: modem_shell: RF_MODEM_LIB_SHMEM_TX_SIZE to 22k

### PR DESCRIPTION
**Pull request by jhirsi**
Increased CONFIG_RF_MODEM_LIB_SHMEM_TX_SIZE to 22k and added a note of PPP baudrate for typical use cases.
Jira items: MOSH-344 and MOSH-352.

### CODE REVIEW

**Review by trantanen — APPROVED** [review]

File: samples/nrf9160/modem_shell/README.rst [hunk 1]
```diff
@@ -970,7 +970,11 @@ After programming the development kit, test it in the Linux environment by perfo
       PPP: started
       mosh:~$
 
-#. Enter command``ppp uartconf`` that results in the following UART configuration:
+#. It has been noticed that the higher baudrates than the default 115200 are performing better with the usual use cases for PPP/dial up.
+   You can set the nRF9160 DK side uart setting for PPP by using MoSh command, for example ``ppp uartconf -b 921600``.
+   Remember to set the corresponding uart setting accordingly from PC side (in this example it is set within the ``pppd`` command later).
```
Comment (peknis) [inline samples/nrf9160/modem_shell/README.rst:975]:
```suggestion
   Higher baudrates than the default 115200 result in better performance with the usual use cases for PPP/dial up.
   Set the nRF9160 DK side UART for PPP with a MoSh command, for example ``ppp uartconf -b 921600``.
   You also need to set the corresponding UART accordingly from PC side (in this example, within the ``pppd`` command).
```
This is not a step, right? I see it rather as additional information between the steps.

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by tokangas — APPROVED** [review]
Looks good. Maybe you could add the reason for the memory size increase to the commit message of 8964dc6, because the Jira ticket is only available internally.

File: samples/nrf9160/modem_shell/README.rst (hunk 1 above)
Comment (jhirsi) [reply samples/nrf9160/modem_shell/README.rst]: I put it as a step to get reader to stop and think here, but removed the step marking anyways now.

**Comment by jhirsi** [comment]
> Looks good. Maybe you could add the reason for the memory size increase to the commit message of [8964dc6]

Done

**Review by hiltunent — APPROVED** [review]

File: samples/nrf9160/modem_shell/README.rst [hunk 2]
```diff
@@ -970,7 +970,11 @@ After programming the development kit, test it in the Linux environment by perfo
       PPP: started
       mosh:~$
 
-#. Enter command``ppp uartconf`` that results in the following UART configuration:
+   Higher baudrates than the default 115200 result in better performance with the usual use cases for PPP/dial up.
+	Set the nRF9160 DK side UART for PPP with a MoSh command, for example ``ppp uartconf -b 921600``.
+	You also need to set the corresponding UART accordingly from PC side (in this example, within the ``pppd`` command).
```
Comment (peknis) [inline samples/nrf9160/modem_shell/README.rst:975]:
```suggestion
   Higher baudrates than the default 115200 result in better performance with the usual use cases for PPP/dial up.
   Set the nRF9160 DK side UART for PPP with a MoSh command, for example ``ppp uartconf -b 921600``.
   You also need to set the corresponding UART accordingly from PC side (in this example, within the ``pppd`` command).
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**